A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -402,10 +396,17 @@ class A320_Neo_MFD_Compass extends NavSystemElement {
         this.svg = this.gps.getChildById("Compass");
         this.svg.aircraft = Aircraft.A320_NEO;
         this.svg.gps = this.gps;
+        const url = document.getElementsByTagName("a320-neo-mfd-element")[0].getAttribute("url");
+        this.disp_index = parseInt(url.substring(url.length - 1));
     }
     onEnter() {
     }
     onUpdate(_deltaTime) {
+        const pot_index = this.disp_index == 1 ? 89 : 91;
```

Comment:
might be good to make the pot_index a property like disp_index too, to avoid this duplication?

Reply:
Done